### PR TITLE
feat: a day in the life of a technical writer

### PR DESCRIPTION
**Pull request by gvasquezvargas**
Blog post on Technical Writing. Structured into: 

- What does a Technical Writer do? 
- What does a Technical Writer need to know? And where do we get that information?
- What does a day in the life of a Technical Writer look like?

### CODE REVIEW

File: content/posts/a-day-in-the-life-of-a-tech-writer/index.md [hunk 1]
```diff
@@ -0,0 +1,78 @@
+---
+authors: ["gvasquezvargas"]
+title: "A Day in the Life of a Technical Writer"
+date: 2023-05-02T12:15:21Z
```
Comment (tillt) [inline content/posts/a-day-in-the-life-of-a-tech-writer/index.md:4]: Adjusting for scheduled publishing date.

```suggestion
date: 2023-05-10T12:00:00+02:00
```

**Review by tillt — APPROVED** [review]
Thanks Gaby - this is great!

File: content/posts/a-day-in-the-life-of-a-tech-writer/index.md (hunk 1 above)
Comment (tillt) [reply content/posts/a-day-in-the-life-of-a-tech-writer/index.md:4]: Once we apply the above, the preview won't work until the day of publishing.

File: content/posts/a-day-in-the-life-of-a-tech-writer/index.md [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+feature_image: feature.png
+---
+
+Let’s start with the basics. 
+
+## What does a Technical Writer do? 
+
+Truth be told, it is difficult to explain.
+
+* I write instruction manuals for advanced computer users - that is what I tell my parents.
```
Comment (msdolbey) [inline content/posts/a-day-in-the-life-of-a-tech-writer/index.md:17]: This is such a great sentence.

**Review by cmastr — DISMISSED** [review]

**Review by msdolbey — DISMISSED** [review]
👍 
I love this one so much. Great!
Save Ferris

**Review by msdolbey — DISMISSED** [review]
👍 
technical writer stuff

**Review by cmastr — APPROVED** [review]

File: content/posts/a-day-in-the-life-of-a-tech-writer/index.md [hunk 3]
```diff
@@ -0,0 +1,78 @@
+---
+authors: ["gvasquezvargas"]
+title: "A Day in the Life of a Technical Writer"
+date: 2023-05-10T12:00:00+02:00
```
Comment (gvasquezvargas) [inline content/posts/a-day-in-the-life-of-a-tech-writer/index.md:4]:
```suggestion
date: 2023-05-17T12:00:00+02:00
```

**Review by tillt — APPROVED** [review]

**Review by Azhovan — APPROVED** [review]

**Pull request merged**